index.js

```diff
@@ -119,4 +121,8 @@ function lookup(path, isExecutable) {
     }
 }
 
+function escapeSpaces(str) {
+    return str.replace(/ /g, '\\ ');
```

Comment:
I couldn't find one in the node APIs after a little searching, though I'm not extremely familiar.